bin/copy-production-modules.js

```diff
@@ -78,6 +78,7 @@ function processPackage( pkgName, context ) {
 	// collect dependencies from various fields
 	const depFields = [ 'dependencies', 'peerDependencies', 'optionalDependencies' ];
 	const pkgDeps = depFields.flatMap( ( type ) => Object.keys( pkgJson[ type ] || {} ) );
+	const optionalPkgDeps = Object.keys( pkgJson.peerDependenciesMeta || [] );
```

Comment:
Is this enough to detect optional peer dependencies? Should we filter for the `optional: true` property first?

Reply:
That makes sense.
I added a filter to check for the property.